procedures/unit-testing-basics.ipf

```diff
@@ -3447,6 +3453,13 @@ Function RunTest(procWinList, [name, testCase, enableJU, enableTAP, enableRegExp
 
 	else
 		// no early return/abort above this point
+		if(!!fixLogName)
+			dfref dfr = GetPackageFolder()
```

Comment:
While there are some places here where we write "dfref" the current convention is "DFREF".
This is related to be consistent with other reference assignments such as WAVE and FUNCREF for example.

Reply:
A simple search returned me 28 examples with "dfref". Maybe we should correct that in the future.

Reply:
I fixed this in 05c5dbb2587a94577124a55c0f05b4f89d73b7a6. If it is preferred I can create a new issue and add this commit to there.